cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -62,7 +62,7 @@ macro(swift_common_standalone_build_config_llvm product)
     set(${product}_NATIVE_LLVM_TOOLS_PATH "${LLVM_TOOLS_BINARY_DIR}")
   endif()
 
-  if(CMAKE_CROSSCOMPILING)
+  if(CMAKE_CROSSCOMPILING AND SWIFT_INCLUDE_TOOLS)
```

Comment:
Won't this still look for `llvm-tblgen` in the `else()` condition?

Reply:
Correct - but, it will find the `llvm-tblgen` from the host build.

Reply:
Spoke with @Rostepher offline, we shouldn't even bother setting up the LLVM_TBLGEN_PATH if we aren't building the tools.